flightcontrol/flightcontrol.json

```
@@ -14,8 +14,10 @@
           "type": "fargate",
           "cpu": 0.25,
           "memory": 0.5,
-          "buildCommand": "yarn rw deploy flightcontrol",
-          "startCommand": "yarn rw deploy flightcontrol --serve",
+          "port": 8911,
+          "installCommand": "cd flightcontrol && yarn install --frozen-lockfile --production=false",
```

```suggestion
          "installCommand": "cd flightcontrol && NODE_ENV=development yarn install --immutable",
```

Ah, I see what's going on. Also, this is yarn 3 so no `production` option. How does the above look?

---

Also, looks like we need to do the same on L29, correct?
```
"buildCommand": "yarn rw deploy flightcontrol web",
```

---

yes on both accounts